examples/activities/act3-functions/act3-functions.ino

```diff
@@ -20,80 +20,49 @@ void setup() {
 
 void loop() {
   // try hovering over these functions!
-  gradientRed(1.5, 10);
+  rainbow(30);
   blinkingPatterns(2, 10);
   fadeInAndOut(1.5, 1.5);
 }
 
-///////////////////////////////////////////////////////////////////////////////
-// Functions
+///////////////////////////////////Functions//////////////////////////////////////////////
 
 /**
  * Parameters:
- *   - "seconds": a number that describes how long you want each "iteration" to be
- *   - "numSteps": a number that describes how many times you want the red color to change
+ *   - "seconds": a number that describes how long you want the rainbow gradient to be
  * ---
- * Function Description: This function sets your LED to white (aka all RGB values to 255) and decreases 
- *   the red RGB value by 10 a "numSteps" amount of times. It will decrease the value every "seconds" seconds.
+ * Function Description: This function rotates your LED through the RGB colors (from red to green to blue) by 
+ *     decreasing each pixel by a value of 1. It will go through the entire rainbow of colors is "seconds" seconds.
  * ---
- * Example Function Call: `gradientRed(2,4)` --> this decreases the red RBG value every 2 seconds, for 4 times
+ * Example Function Call: `rainbowStep(30)` --> creates a rainbow that lasts 30 seconds. 
  */
-void gradientRed(float seconds, int numSteps) {
-  int Red = 250;
-  int Green = 250;
-  int Blue = 250;
-
-  for (int step = 0; step < numSteps; step++) {
-    Red -= 50;
+void rainbow(float seconds) {
+  int Red = 0; 
+  int Green = 0; 
+  int Blue = 0;
+  
+  // Adjust the Red pixel
+  for (Red = 255; Red > 0; Red --) {
+    Green += 1; 
     pixels.setPixelColor(0, pixels.Color(Red, Green, Blue));
     pixels.show();   // Send the updated pixel colors to the hardware.
-    delay(DELAYVAL * 2 * seconds); // Pause before next pass through loop
+    delay(DELAYVAL * 2 * seconds / 765); // Pause before next pass through loop (milliseconds)
```

Comment:
why is it 765? Cool that you were able to figure out the exact value here but explaining this number would be awesome!

Reply:
I'll leave this as a note in the new repo and close this PR for now :)